src/pilot/discover_nodes/discover_nodes.py

```diff
@@ -473,7 +470,7 @@ def is_idrac(client):
     requests_logger.disabled = True
 
     try:
-        doc = client.client.enumerate(DCIM_iDRACCardView)
+        return client.client.is_idrac_ready()
```

Comment:
We don't want to return a value indicating if the idrac is ready or not.  We want to return True if the is_idrac_ready call does not throw an exception regardless of if it returns True or False.

Reply:
Updated code in is_idrac() method for return value as suggested and also tested the code changes with the suggested approach.
Removed the lines which are referencing to DDCIM_iDRACCardView in comments.